test/bin/ci-travis-all.sh

```diff
@@ -0,0 +1,3 @@
+grunt test:nodeunit
```

Comment:
No point duplicating the nodeunit tests in the browserstack run. (If you're only doing that get the concat task, you can just do `grunt concat` directly. have a look in the gruntfile to see what tasks do what)

Reply:
@SimonWoolf  True, but browser tests failed if not preceded by nodeunit tests because of https://github.com/ably/ably-js/issues/462

Reply:
No, `grunt concat` doesn't run the minifier, so is unaffected by #462.